src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -373,6 +369,8 @@ class MoreInfoLight extends LitElement {
       return;
     }
 
+    this._wvSliderValue = wv;
```

Comment:
Shouldn't this be set after `wv = Math.min(255, Math.round((wv * 255) / 100));` ?

Reply:
No, the slider is 0-100, the value we need to sent with the service call is 0-255

Reply:
But this is wrong... so thanks for pointing me to that 😄